someSong/src/pages/browse-questions/browse-questions.ts

```diff
@@ -44,16 +44,15 @@ export class BrowseQuestionsPage {
   getAllEnabledQuestions(){
     console.log(this.questions);
     var enabledQuestions = [];
-    var filteredGenresKeys = this.arrayGenresToKeysGenres(this.filteredGenres);
-    console.log(filteredGenresKeys);
+ //   var filteredGenresKeys = this.arrayGenresToKeysGenres(this.filteredGenres);
+ //   console.log(filteredGenresKeys);
     for (var i =0; i<this.questions.length; i++){
-      var b = this.findOne(this.questions[i]["genres"],filteredGenresKeys);
+ //     var b = this.findOne(this.questions[i]["genres"],filteredGenresKeys);
       console.log(this.questions[i]["genres"]);
-      console.log(filteredGenresKeys);
-      console.log(b);
-      if ( !b){
+ //     console.log(filteredGenresKeys);
+//      if ( !b){
         enabledQuestions.push(this.questions[i]);
```

Comment:
why do we even have this function?
why not just use the `getAllUnresolvedQuestions` in the `Question` provider?
why add the `enable=true` property in the constructor, and return a new array of enabled questions?
doesn't it have the same result as `this._quetion.getAllUnresolvedQuestions`?